src/TriggerActivityMakerTrackFinding.cpp

```diff
@@ -0,0 +1,303 @@
+/**
+ * @file TriggerActivityMakerTrackFinding.cpp
+ *
+ * This is part of the DUNE DAQ Application Framework, copyright 2021.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#include "triggeralgs/TrackFinding/TriggerActivityMakerTrackFinding.hpp"
+#include "TRACE/trace.h"
+#define TRACE_NAME "TriggerActivityMakerTrackFindingPlugin"
+#include <vector>
+#include <math.h>
+
+using namespace triggeralgs;
+
+void
+TriggerActivityMakerTrackFinding::operator()(const TriggerPrimitive& input_tp,
+					     std::vector<TriggerActivity>& output_ta)
+{
+
+  // Add useful info about recived TPs here for FW and SW TPG guys.
+  if (m_print_tp_info){
+    TLOG(1) << " ########## m_current_window is reset ##########\n"
+	    << " TP Start Time: " << input_tp.time_start << ", TP ADC Sum: " << input_tp.adc_integral
+	    << ", TP TOT: " << input_tp.time_over_threshold << ", TP ADC Peak: " << input_tp.adc_peak
+	    << ", TP Offline Channel ID: " << input_tp.channel << "\n";
+  }
+  
+  // 0) FIRST TP =====================================================================
+  // The first time operator() is called, reset the window object.
+  if (m_current_window.is_empty()) {
+    m_current_window.reset(input_tp);
+    m_primitive_count++;
+    return;
+  }
+  
+  // If the difference between the current TP's start time and the start of the window
+  // is less than the specified window size, add the TP to the window.
+  bool adj_pass = 0; // sets to true when adjacency logic is satisfied
+  bool window_filling = 0; // sets to true when window is ready to test the adjacency logic
```

Comment:
I guess this is more of a `window_filled`

Reply:
This is correct name: this parameter is true when window filling is ongoing, and is false when window is complete.

Reply:
Agree, changed to `window_filled`